components/webrtc/notification/webrtc_notification.jsx

```diff
@@ -84,35 +83,24 @@ export default class WebrtcNotification extends React.Component {
     }
 
     onIncomingCall(incoming) {
-        if (this.mounted) {
+        if (this.mounted && this.props.enableWebrtc) {
             const userId = incoming.from_user_id;
-            const userMedia = navigator.getUserMedia || navigator.webkitGetUserMedia || navigator.mozGetUserMedia;
-            const featureEnabled = Utils.isFeatureEnabled(PreReleaseFeatures.WEBRTC_PREVIEW);
-
-            if (featureEnabled) {
-                if (WebrtcStore.isBusy()) {
-                    WebSocketClient.sendMessage('webrtc', {
-                        action: WebrtcActionTypes.BUSY,
-                        from_user_id: UserStore.getCurrentId(),
-                        to_user_id: userId,
-                    });
-                    this.stopRinging();
-                } else if (userMedia) {
-                    WebrtcStore.setVideoCallWith(userId);
-                    this.setState({
-                        userCalling: UserStore.getProfile(userId),
-                    });
-                } else {
-                    WebSocketClient.sendMessage('webrtc', {
-                        action: WebrtcActionTypes.UNSUPPORTED,
-                        from_user_id: UserStore.getCurrentId(),
-                        to_user_id: userId,
-                    });
-                    this.stopRinging();
-                }
+
+            if (WebrtcStore.isBusy()) {
+                WebSocketClient.sendMessage('webrtc', {
+                    action: WebrtcActionTypes.BUSY,
+                    from_user_id: UserStore.getCurrentId(),
+                    to_user_id: userId,
+                });
+                this.stopRinging();
+            } else if (Utils.isUserMediaAvailable()) {
+                WebrtcStore.setVideoCallWith(userId);
+                this.setState({
+                    userCalling: UserStore.getProfile(userId),
+                });
             } else {
                 WebSocketClient.sendMessage('webrtc', {
-                    action: WebrtcActionTypes.DISABLED,
```

Comment:
Is there any use case for retaining the `DISABLED` message? I guess if it's disabled at the server level, it's not likely to trigger a message in the first place?

Reply:
Yeah, I think you're right. I'll update this.

Reply:
Done update.